docs/user/localizing-your-projects.rst

```diff
@@ -69,9 +69,14 @@ following required fields:
 5. **Download prefix**: used for manually downloading files. Select any
    localizable file on GitHub, click ``Raw`` and replace locale code and the
    following bits in the URL with {locale_code}.
-6. Click **SAVE PROJECT** at the bottom of the page.
-7. After the page reloads, click **SYNC** and wait for Pontoon to import
-   strings. You can monitor the progrss in the Sync log (``/sync/log/``).
+7. Click **SAVE PROJECT** at the bottom of the page.
+8. All new projects in Pontoon are private by default and aren't visible to
+   localizers and locale managers.
```

Comment:
This is not an actual step, so I would instead append it to the last step.

Reply:
I moved it to the last step, can you check if the phrase still sounds okay?